webpack.config.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
```

Comment:
- [ ] I can't run your project, as for now you need to install the `webpack dev server `, you can use this command `npm i webpack-dev-server`. then configure the WebPack to run your app

Reply:
![Screenshot (5)](https://user-images.githubusercontent.com/102296263/171996624-88cd5e2d-ab0c-4552-ab3f-27b86f85855b.png)

Reply:
always see this I worked for about 12 hours and I cant fix it